src/data/complex/basic.lean

```diff
@@ -21,6 +23,17 @@ namespace complex
 
 noncomputable instance : decidable_eq ℂ := classical.dec_eq _
 
+/-- The equivalence between the complex numbers and `ℝ × ℝ`. -/
+def real_prod_equiv : ℂ ≃ (ℝ × ℝ) :=
```

Comment:
I wonder if this should move later in the file, and become a `≃+`, or even a `≃ₗ[ℝ]`.

Reply:
But maybe these require too many imports.

Reply:
They do require new imports, so I'd rather see them added in another file / PR.

Reply:
Also, shouldn't the name be swapped? `equiv_real_prod`?

Reply:
Oh yeah, fixed.